gulpfile.js/tasks/html.js

```diff
@@ -24,14 +24,17 @@ var htmlTask = function() {
     return JSON.parse(fs.readFileSync(dataPath, 'utf8'))
   }
 
+  var customFilters = TASK_CONFIG.html.customFilters || function(env) {}
```

Comment:
do you need the no-op function fallback? Does it error if customFilters is undefined or just ignore it? Would `false` also work? Not a big deal, just slightly simpler.